src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -156,6 +156,9 @@ private void nextContainer() throws IOException, IllegalArgumentException,
             final SAMSequenceRecord sequence = cramHeader.getSamFileHeader()
                     .getSequence(container.sequenceId);
             refs = referenceSource.getReferenceBases(sequence, true);
+            if (refs == null) {
+                throw new RuntimeException(String.format("Contig %s not found in the reference file.", sequence.getSequenceName()));
```

Comment:
This class uses `RuntimeException` everywhere else -- it would be inconsistent to use a more specific exception here.